ui/js/controllers/main.js

```diff
@@ -508,10 +508,13 @@ treeherderApp.controller('MainCtrl', [
         };
 
         const getNewReloadTriggerParams = function () {
-            return _.pick(
-                $location.search(),
-                ThResultSetStore.reloadOnChangeParameters,
-            );
+            const locationSearch = $location.search();
+            return Object.assign(
```

Comment:
I think a reduce might be clearer here:

    ThResultSetStore.reloadOnChangeParameters.reduce((acc, prop) => ({
              ...acc, [prop]: locationSearch[prop],
                }), {});